### PR TITLE
Flag gas abs

### LINKED ISSUE #5: Include flag to remove gaseous absorption

### PR DESCRIPTION
**Pull request by Alteck76**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,7 +35,7 @@ F90FLAGS = -module $(mod) -fpp -qopenmp -g -traceback -check bounds -debug exten
 
 # Let's start with setting up some paths
 # ---------------------------------------------------------------------------------------------------------------------------------------
-PATH_S3COM = $(HOME)/github/S3COM
+PATH_S3COM = $(HOME)/S3COM-test/S3COM
```
Comment (odrans) [inline Makefile]: Would be nice to find an easier solution than change the path back and forth between our commits

File: src/io/io_s3com.f90 [hunk 2]
```diff
@@ -101,7 +101,7 @@ subroutine write_output_rad(s3com, model, nml)
     character(LEN = 256) :: fn_out_rad, suffix, attr_instrument
 
     suffix = trim(nml%suffix_out)
-    if(trim(suffix) .ne. "") suffix = "_"//trim(suffix)//"_"
+    if(trim(suffix) .ne. "") suffix = "_"//trim(suffix)//""
```
Comment (odrans) [inline src/io/io_s3com.f90]: Cleaner to just remove //""

**Review by odrans — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Alteck76) [reply Makefile]: Yes, I will rename my path to yours.

Comment (Alteck76) [reply Makefile]: Done!

File: src/io/io_s3com.f90 (hunk 2 above)
Comment (Alteck76) [reply src/io/io_s3com.f90]: Done!

**Comment by odrans** [comment]
Perfect! Thanks!

**Pull request merged**